tcadmin/tests/test_current_clients.py

```diff
@@ -52,8 +52,9 @@ async def test_fetch_clients_empty(AuthForClients):
     assert list(resources) == []
 
 
+@pytest.mark.usefixtures("appconfig")
 @pytest.mark.asyncio
-async def test_fetch_clients_managed(AuthForClients, make_client):
+async def test_fetch_clients_managed(AuthForClients, make_client, appconfig):
```

Comment:
Only one of these two approaches (usefixtures or listing in the function arguments) is necessary, not both.

Even easier, the fixture can be enabled for a whole module by putting

```python
pytestmark = pytest.mark.usefixtures("appconfig")
```

near the top of the module.

I'd suggest doing that for all modules where any test case needs the fixture.  If there are modules where some test cases need the fixture but others fail with the fixture, then a case-by-case addition of `appconfig` to the function arguments.

Reply:
Oh, I will try the new approach

Reply:
This worked for all, there was no need to add `appconfig` as an argument anywhere.